package.xml

```diff
@@ -28,6 +28,7 @@
   <test_depend>launch_testing</test_depend>
   <test_depend>launch_testing_ament_cmake</test_depend>
   <test_depend>rmw_implementation_cmake</test_depend>
+  <test_depend>rosgraph_msgs</test_depend>
```

Comment:
remove? or maybe part of future tests?

Reply:
https://github.com/ros2/domain_bridge/blob/348fd60576b841e2ed5fb24353b9205d738aad99/examples/example_bridge_config.yaml#L8
https://github.com/ros2/domain_bridge/blob/348fd60576b841e2ed5fb24353b9205d738aad99/test/test_domain_bridge.launch.py#L63-L64

I found that the dependency was missing while working with a partially-built ros2 workspace as a base.

Reply:
ah, good catch.